agent/src/index.ts

```diff
@@ -521,6 +522,7 @@ export async function createAgent(
         // character.plugins are handled when clients are added
         plugins: [
             bootstrapPlugin,
+            webSearchPlugin,
```

Comment:
should we be putting it behind a secret/env flag?

Reply:
Iirc, plugins are generally handled in character files, since each agent will likely use different plugins. Would we prefer to change that? Doesn't seem necessary to me unless 'which plugin does this agent have' ought to be a secret